ossdbtoolsservice/edit_data/templates/pg_templater.py

```diff
@@ -0,0 +1,46 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from .templater import Templater
```

Comment:
Where this template existing previously?

Reply:
Earlier they were using as it is in implementation. Now all the templates are collected in a templater class which is being called to get the template format.